src/routes/docs/quick-starts/apple/+page.markdoc

```diff
@@ -59,19 +59,23 @@ When dependency resolution is complete, click **Add Package** again to add the S
 In order to allow creating OAuth sessions, the following URL scheme must be added to your **Info.plist** file.
 
 ```xml
-<key>CFBundleURLTypes</key>
-<array>
+<plist version="1.0">
```

Comment:
We don't need the plist definition here, since the intention is to add this block to an existing plist, not create a new one